docs/team-process.md

```diff
@@ -9,29 +9,29 @@ Here are some few resources for documentation associated with onboarding:
 1. [Onboarding checklist](https://docs.google.com/document/d/1zhvp8vcWnVSvUbTKZ-0sHrf5lA6OTlpXqvjeLgKJtVk)
 2. [Onboarding Reference Guide](https://docs.google.com/document/d/1jOd0wAsqTDBKsFeEvUj9ezvdTimTsqTwkYd6Imibl3c)
 
-If you do not have permissions to view these documents, please reaach out to the scrum master. 
+If you do not have permissions to view these documents, please reaach out to the scrum master.
```

Comment:
I know this isn't your typo, but I just noticed:

`reaach` should be `reach`

Reply:
...and now that I'm looking even closer, do we need to change "scrum master" to something else?

Reply:
changed...thanks Jim